### PR TITLE
⚡️ Load class metadata in chunk

### PR DESCRIPTION
**Pull request by williamchong**

### CODE REVIEW

File: src/server/api/routes/nft/metadata.js [hunk 1]
```diff
@@ -19,45 +19,19 @@ router.get('/nft/metadata', async (req, res, next) => {
...
-    res.set('Cache-Control', 'public, max-age=1');
-    res.json({
-      classData,
-      iscnData,
-      ownerInfo,
-      listings,
-      purchaseInfo,
-    });
+    if (Array.isArray(classId)) {
+      const data = await Promise.all(classId.map(id => getNFTClassData(id)));
```
Comment (WeiJun0827) [inline src/server/api/routes/nft/metadata.js:23]: Do we need to guard the size of `classId`?

File: src/store/modules/nft.js [hunk 2]
```diff
@@ -413,6 +414,82 @@ const actions = {
...
+            info: purchaseInfo,
+          });
+        }
+        if (classData.iscn_owner) {
+          const userPromise = dispatch(
+            'lazyGetUserInfoByAddress',
+            classData.iscn_owner
+          );
+          userPromises.push(userPromise);
+        }
```
Comment (WeiJun0827) [inline src/store/modules/nft.js:454]: Maybe we can refactor this part, and use it in single and multiple `classId` cases.
Or let API handle multiple `classId` only, so we don't need to handle the single class case?

File: src/store/modules/nft.js [hunk 3]
```diff
@@ -413,6 +414,82 @@ const actions = {
...
+        const userPromise = dispatch(
+          'lazyGetUserInfoByAddress',
+          classData.iscn_owner
+        );
+        // Need to await if the action fires in during SSR
+        if (!process.client) await userPromise;
+      }
+    }
+  },
+  async fetchNFTClassesAggregatedInfo({ commit, dispatch }, classId) {
```
Comment (WeiJun0827) [inline src/store/modules/nft.js:492]: Do we still keep `fetchNFTClassesAggregatedInfo` here?

Comment (williamchong) [reply src/store/modules/nft.js:492]: yes this is extra

**Review by WeiJun0827 — APPROVED** [review]

**Pull request merged**